wa/instruments/trace_cmd.py

```diff
@@ -125,6 +125,12 @@ class TraceCmdInstrument(Instrument):
                   value by going down from the specified size in
                   ``buffer_size_step`` intervals.
                   """),
+        Parameter('top_buffer_size', kind=int, default=None,
+                  global_alias='trace_top_buffer_size',
+                  description="""
+                  The same as buffer_size except sets the top-level buffer
+                  size instead of the devlib buffer size.
```

Comment:
Would you be able to add an extra sentence as you mentioned in the devlib Pr to say that if not set, the top buffer size will default to the same as `buffer_size`?

Reply:
No problem, done